app/Integrations/Packagist/Packages.php

```diff
@@ -171,7 +171,7 @@ public function download(array $package)
         $meta = stream_get_meta_data($temp);
         $path = $meta['uri'];
 
-        ($zip = new ZipArchive())->open($path);
+        with($zip = new ZipArchive())->open($path);
```

Comment:
For the 5.6 version we're testing with on Travis, we do.

https://travis-ci.org/CachetHQ/Cachet/jobs/153497870